statistics/selectivity.go

```diff
@@ -499,7 +499,7 @@ func getMaskAndRanges(ctx sessionctx.Context, exprs []expression.Expression, ran
 	switch rangeType {
 	case ranger.ColumnRangeType:
 		accessConds = ranger.ExtractAccessConditionsForColumn(exprs, cols[0])
-		ranges, err = ranger.BuildColumnRange(accessConds, ctx, cols[0].RetType, types.UnspecifiedLength)
+		ranges, accessConds, _, err = ranger.BuildColumnRange(accessConds, ctx, cols[0].RetType, types.UnspecifiedLength, ctx.GetSessionVars().RangeMaxSize)
```

Comment:
Why ignore the `remainedConds` here?

Reply:
`ExtractAccessConditionsForColumn` extracts `accessConds` from `exprs` and ignores those remaining conditions. For the same reason, After `BuildColumnRange`, we only cares about `accessConds` and can ignore `remainedConds`.